Analyze.default.props

```
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  Copyright (c) Microsoft Corporation
+  SPDX-License-Identifier: MIT
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003" TreatAsLocalProperty="Platform">
+  <PropertyGroup Condition="'$(Analysis)'=='True'">
+    <RunCodeAnalysis>true</RunCodeAnalysis>
+    <DisableAnalyzeExternal>true</DisableAnalyzeExternal>
+    <CodeAnalysisRuleSet>$(SolutionDir)\Analyze.default.ruleset</CodeAnalysisRuleSet>
```

```suggestion
    <CodeAnalysisRuleSet>$(SolutionDir)Analyze.default.ruleset</CodeAnalysisRuleSet>
```
SolutionDir always ends in a \ already